_redirects

```diff
@@ -0,0 +1,20 @@
+/about             /learn-more/#about
+/about.html        /learn-more/#about
+/community         /learn-more/#get-involved
+/community.html    /learn-more/#get-involved
+/dask              https://docs.dask.org/en/stable/gpu.html
+/dask.html         https://docs.dask.org/en/stable/gpu.html
+/hpc               https://docs.rapids.ai/deployment/stable/hpc.html
+/hpc.html          https://docs.rapids.ai/deployment/stable/hpc.html
+/hpo               https://docs.rapids.ai/deployment/stable/examples/index.html
+/hpo.html          https://docs.rapids.ai/deployment/stable/examples/index.html
+/pip               https://docs.rapids.ai/install#pip
+/pip.html          https://docs.rapids.ai/install#pip
+/plotly            https://docs.rapids.ai/visualization
+/plotly.html       https://docs.rapids.ai/visualization
+/start             https://docs.rapids.ai/install
```

Comment:
What about the other redirects from the old site? https://github.com/rapidsai/rapids.ai/blob/3528643be36219467755e11a8ac7a2a1fd715fe9/_redirects

The Slack invite is the one I want the most. I use that shortened URL to point to our Slack community pretty often.

```suggestion
/slack-invite      https://join.slack.com/t/rapids-goai/shared_invite/zt-trnsul8g-Sblci8dk6dIoEeGpoFcFOQ
/start             https://docs.rapids.ai/install
```

Reply:
Goooood catch. I will push an update with these changes.

Reply:
Included in bb59d2b.